client/src/views/asset-list.js

```diff
@@ -15,30 +40,116 @@ export default ({ assetMap, t, ...S }) => {
           <h1>{t`Registered assets`}</h1>
         </div>
       </div>
-
       <div className="container">
-        { !assets.length ? <p>{t`No registered assets`}</p>
-        : <div className="assets-table">
+        { !assetList ?  process.browser 
+            ? <div className="load-more-container"><img src="img/Loading.gif" /></div>
+            : <div className="load-more-container">
```

Comment:
Why does this need to render an error here? If the request to load the assets fails, displaying an error should already be dealt with (via the `error$` stream).

I think this could be simplified to always show the loading indicator when the asset list isn't available, for both server and client side rendering (the server-side renderer waits for everything to load before capturing the html, so the loading indicator won't be visible).

Reply:
changes have been made